pkg/operators/catalogsource_types.go

```diff
@@ -29,6 +29,11 @@ type CatalogSourceSpec struct {
 	// SourceType is the type of source
 	SourceType SourceType
 
+	// Priority field assigns a weight to the catalog source to prioritize them so that it can be consumed by the dependency resolver.
+	// Higher weight indicates that this catalog source is preferred over lower weighted catalog sources during dependency resolution.
+	// +Optional
+	Priority int `json:"priority,omitempty"`
+
 	// ConfigMap is the name of the ConfigMap to be used to back a configmap-server registry.
 	// Only used when SourceType = SourceTypeConfigmap or SourceTypeInternal.
 	// +Optional
```

Comment:
Slightly derivative but, is this file the "latest" version of this type? Should we be manually modifying it at all? Or should there be some sync target in the makefile?

Reply:
I will have a follow-up PR to remove these files as discussed in [slack](https://coreos.slack.com/archives/G3T7N42NP/p1595340592091300)

Reply:
might as well delete these from this PR.